capstone/src/main/webapp/script.js

```diff
@@ -136,13 +158,14 @@ function configureAllCardsHTML(cards) {
 function showAllButton() {
   var allCardsContainer = document.getElementById("all-cards-container");
   if (allCardsContainer.style.display == "none") {
-    allCardsContainer.style.display = "block";
-  } 
-  else {
+    allCardsContainer.style.display = "flex";
```

Comment:
Looks like materialize defines a `.hide` class: https://materializecss.com/helpers.html. Update this function to add or remove the `.hide` class rather than directly setting the `display` property. That way you don't don't to keep the `display` property in sync between your CSS and JS files.